sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -799,16 +810,21 @@ def test_check_estimator_transformer_no_mixin():
 
 def test_check_estimator_clones():
     # check that check_estimator doesn't modify the estimator it receives
-    from sklearn.datasets import load_iris
 
-    iris = load_iris()
+    X, y = make_classification(
+        n_samples=50,
+        n_classes=3,
+        n_informative=3,
+        n_clusters_per_class=1,
+        n_features=5,
+        random_state=0,
+    )
 
     for Estimator in [
         GaussianMixture,
         LinearRegression,
         SGDClassifier,
         PCA,
-        ExtraTreesClassifier,
```

Comment:
removing this cause it's very slow. Removing this reduces the time of this test on my machine from over 7s to 1.7s

Reply:
Having it doesn't really add anything for the purpose of the test anyway. So I'm happy to have it gone.